igor-core/igor-core.gradle

```diff
@@ -1,6 +1,7 @@
 dependencies {
   implementation "com.netflix.spinnaker.kork:kork-artifacts"
   implementation "com.netflix.spinnaker.kork:kork-core"
+  api "com.netflix.spinnaker.kork:kork-plugins"
```

Comment:
I think this should be `implementation` and only under `igor-web` until we write an `ExtensionPoint`.

Reply:
That makes sense. I think we were just being consistent with how it was done in the other services.